### PR TITLE
Show Welcome modal: only on homepage, only once

### PR DESCRIPTION
**Pull request by erikphansen**
> I'm still mentally walking through this to make sure this will work as expected in all cases.

## Description
Add a Welcome To The New VA modal that only appears on the homepage and only appears once.

## Testing done
Local testing

### Updated screenshot
image

### Old screenshot
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by DanielleSoCompany** [comment]
Hi @erikphansen. I think this looks great. My only question is whether this one is supposed to have the "+ Vets.gov" in the header, if this is being shown to every viewer and not just those being redirected from Vets.gov. I may be totally off here, so if this is correct, please feel free to ignore this comment. Just wanted to raised as a question in case. Thanks!

**Comment by erikphansen** [comment]
Thanks @DanielleSoCompany. Good point. Do we have an existing VA.gov asset that's suitable for this modal?

**Comment by DanielleSoCompany** [comment]
I'm not sure. @amycesal @emilywaggoner, would you be able to help answer Erik's question above?

File: src/platform/site-wide/announcements/components/WelcomeToNewVAModal.jsx
Comment (jbalboni) [inline src/platform/site-wide/announcements/components/WelcomeToNewVAModal.jsx]: Why don't we want this to run in preview?

Comment (erikphansen) [reply src/platform/site-wide/announcements/components/WelcomeToNewVAModal.jsx]: Good call. Will fix.

**Comment by amycesal** [comment]
So this VA.gov + Vets.gov lock-up was intended for people who are coming from vets.gov. 
See this issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12060

Will the modal appear for everyone coming to the new va site, not just from vets? It might make sense to use the VA logo + seal. Or maybe we don't an image/logo.

**Comment by erikphansen** [comment]
@amycesal this modal will be show the only once to visitors who navigate directly to the va.gov homepage. You will not see it if you are redirected from vets.gov. If we already have an excisting image asset to use I can probably just drop it in place of what is shown in the screenshot. Otherwise I'll need a new asset to use. Someone in design or product needs to make that call. Thank you!

**Comment by erikphansen** [comment]
I am also in favor of calling all modals "lock ups" from here on out.

**Review by ncksllvn — APPROVED** [review]
This seems like a case we could just use the VA logo. But either way, LGTM! After merging, you can test this by going to https://dev.vets.gov and making sure that the Vets-to-VA modal shows after the redirect, and then again by navigating directly.

**Comment by amycesal** [comment]
Yes, makes sense to use the VA logo here. Let me know if you need the image file, I can find one for you. 

The lockup I was referring to is the combination of va + vets.gov. 2 parts of a logo, like an icon and the wordmark, or the main company name and the subtitle are often called lockups. <- design speak

**Pull request merged**
